index.js

```diff
@@ -472,63 +463,82 @@ io.on('connection', (socket) => {
-    await sendTwilioMessage(locationData.twilio_number, session.phonenumber, `Please check on the bathroom. Please respond with 'ok' once you have checked on it.`);
-    await db.startChatbotSessionState(session);
-    setTimeout(reminderMessage, locationData.unrespondedTimer, session.locationid);
+  console.log("Intial message sent");
+  var location = session.locationid;
+  var alertReason = session.alert_reason;
+  locationData = await db.getLocationData(location);
+  await sendTwilioMessage(locationData.twilio_number, session.phonenumber, `This is a ${alertReason} alert. Please check on the bathroom. Please respond with 'ok' once you have checked on it.`);
+  await db.startChatbotSessionState(session);
+  setTimeout(reminderMessage, locationData.unresponded_timer, session.sessionid);
+  setTimeout(fallbackMessage, locationData.unresponded_session_timer, session.sessionid)
 }
 
-async function reminderMessage(location) {
-    console.log("Reminder message being sent");
-    let session = await db.getMostRecentSession(location); // Gets the updated state for the chatbot
+async function reminderMessage(sessionid) {
+  console.log("Reminder message being sent");
+  let session = await db.getSessionWithSessionId(sessionid); // Gets the updated state for the chatbot
+  if(session.chatbot_state == STATE.STARTED) {
+    //Get location data
+    var location = session.locationid;
     locationData = await db.getLocationData(location)
-    if(session.chatbot_state == 'Started') {
-        //send the message
-        await sendTwilioMessage(locationData.twilio_number, session.phonenumber, `This is a reminder to check on the bathroom`)
-        session.chatbot_state = 'Waiting for Response';
-        await db.saveChatbotSession(session);
-    }
-    //else do nothing
+    //send the message
+    await sendTwilioMessage(locationData.twilio_number, session.phonenumber, `This is a reminder to check on the bathroom`)
+    session.chatbot_state = STATE.WAITING_FOR_RESPONSE;
+    let chatbot = new Chatbot(session.sessionid, session.locationid, session.chatbot_state, session.phonenumber, session.incidenttype, session.notes);
+    await db.saveChatbotSession(chatbot);
+  }
+  //else do nothing
 }
 
-//Heartbeat Helper Functions
+async function fallbackMessage(sessionid) {
+  console.log("Fallback message being sent");
+  let session = await db.getSessionWithSessionId(sessionid); // Gets the updated state for the chatbot
+  if(session.chatbot_state == STATE.WAITING_FOR_RESPONSE) {
+    console.log("Fallback if block");
+    let locationData = await db.getLocationData(session.locationid)
+    console.log(`fallback number is:  ${locationData.fallback_phonenumber}`)
+    console.log(`twilio number is:  ${locationData.twilio}`)
+    await sendTwilioMessage(locationData.twilio_number, locationData.fallback_number,`An alert to check on the washroom at ${locationData.location_human} was not responded to. Please check on it`)
```

Comment:
When was `fallback_number` added to the DB? I don't see it in any of the migration scripts.